### PR TITLE
Add StructureDefinition Annotation to resources

### PR DESCRIPTION
**Pull request by KevinCranmer**

### CODE REVIEW

File: argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/Conformance.java [hunk 1]
```diff
@@ -42,6 +43,7 @@
         "${dstu2.conformance:gov.va.api.health.dstu2.api.swaggerexamples"
             + ".SwaggerConformance#conformance}")
 @DoesNotHaveTargetPatientQueryParameter
+@StructureDefinition("http://hl7.org/fhir/DSTU2/conformance.html")
```
[the remark this answers is not in the document]
Comment (jhulbert-va) [reply argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/Conformance.java:46]: Yes. We're super likely to add a resource in Yanartas without thinking about adding the url here. If it currently exists, the new annoation should be added.

File: argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/OperationOutcome.java [hunk 2]
```diff
@@ -36,6 +37,7 @@
         "${dstu2.operationOutcome:gov.va.api.health.dstu2.api.swaggerexamples"
             + ".SwaggerOperationOutcome#operationOutcome}")
 @DoesNotHaveTargetPatientQueryParameter
+@StructureDefinition("https://www.hl7.org/fhir/operationoutcome.html")
```
Comment (jhulbert-va) [inline argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/OperationOutcome.java:40]: This doesn't look resource specific.

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Coverage.java [hunk 3]
```diff
@@ -54,6 +55,7 @@
     description = "https://www.hl7.org/fhir/R4/coverage.html",
     example = "${r4.coverage:gov.va.api.health.r4.api.swaggerexamples.SwaggerCoverage#coverage}")
 @HasTargetPatientQueryParameter({"beneficiary", "patient"})
+@StructureDefinition("http://hl7.org/fhir/us/core/StructureDefinition/us-core-coverage")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Coverage.java:58]: I believe coverage was CARIN-BB

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/CoverageEligibilityResponse.java [hunk 4]
```diff
@@ -62,6 +63,7 @@
     fields = {"request", "_request"},
     message = "Exactly one request value must be specified.")
 @HasTargetPatientQueryParameter("patient")
+@StructureDefinition("https://www.hl7.org/fhir/coverageeligibilityresponse.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/CoverageEligibilityResponse.java:66]: CARIN-BB

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/ExplanationOfBenefit.java [hunk 5]
```diff
@@ -62,6 +63,7 @@
         "${r4.explanationOfBenefit:gov.va.api.health.r4.api.swaggerexamples"
             + ".SwaggerExplanationOfBenefit#explanationOfBenefit}")
 @HasTargetPatientQueryParameter("patient")
+@StructureDefinition("https://www.hl7.org/fhir/endpoint.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/ExplanationOfBenefit.java:66]: This looks like the wrong structure definition.

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/InsurancePlan.java [hunk 6]
```diff
@@ -56,6 +57,7 @@
         "${r4.insurancePlan:gov.va.api.health.r4.api.swaggerexamples."
             + "SwaggerInsurancePlan#insurancePlan}")
 @DoesNotHaveTargetPatientQueryParameter
+@StructureDefinition("https://www.hl7.org/fhir/R4/insuranceplan.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/InsurancePlan.java:60]: CARIN-BB

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/OperationOutcome.java [hunk 7]
```diff
@@ -36,6 +37,7 @@
         "${r4.operationOutcome:gov.va.api.health.r4.api.swaggerexamples"
             + ".SwaggerOperationOutcome#operationOutcome}")
 @DoesNotHaveTargetPatientQueryParameter
+@StructureDefinition("https://www.hl7.org/fhir/operationoutcome.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/OperationOutcome.java:40]: Doesn't look resource specific.

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Endpoint.java [hunk 8]
```diff
@@ -49,6 +50,7 @@
     example =
         "${r4.endpoint:gov.va.api.health.r4.api.swaggerexamples." + "SwaggerEndpoint#endpoint}")
 @DoesNotHaveTargetPatientQueryParameter
+@StructureDefinition("https://www.hl7.org/fhir/endpoint.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Endpoint.java:53]: This doesn't look resource specific.

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Person.java [hunk 9]
```diff
@@ -51,6 +52,7 @@
     description = "https://www.hl7.org/fhir/R4/person.html",
     example = "${r4.person:gov.va.api.health.r4.api.swaggerexamples.SwaggerPerson#person}")
 @HasTargetPatientQueryParameter("patient")
+@StructureDefinition("https://www.hl7.org/fhir/person.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Person.java:55]: Doesn't look version specific.

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/TerminologyCapabilities.java [hunk 10]
```diff
@@ -34,6 +35,7 @@
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 @Schema(description = "https://www.hl7.org/fhir/R4/terminologycapabilities.html")
 @DoesNotHaveTargetPatientQueryParameter
+@StructureDefinition("https://hl7.org/fhir/terminologycapabilities.html")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/TerminologyCapabilities.java:38]: Version?

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/CoverageEligibilityResponse.java (hunk 4 above)
Comment (KevinCranmer) [reply us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/CoverageEligibilityResponse.java:66]: I'm not seeing CoverageEligibilityResponse anywhere in https://hl7.org/fhir/us/carin-bb/toc.html

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/InsurancePlan.java (hunk 6 above)
Comment (KevinCranmer) [reply us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/InsurancePlan.java:60]: Wasn't able to find insurancePlan in https://hl7.org/fhir/us/carin-bb/toc.html either

**Review by jhulbert-va — APPROVED** [review]

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**